v22.2/use-a-local-file-server.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: Use a Local File Server for Bulk Operations
-summary: Learn how to create a simple file server for use with bulk operations within CockroachDB
+title: Use a Local File Server
+summary: Learn how to create a simple file server for use with operations within CockroachDB
```

Comment:
The end of this summary is a little awkward, do you mind changing it while you're working here? Perhaps, something like the following:

> Learn how to create a simple file server to use with CockroachDB operations.